packages/react-storage/src/components/StorageBrowser/Views/Controls/ActionSelect.tsx

```diff
@@ -24,50 +27,110 @@ interface Action {
   type: string;
 }
 
-type RenderActionItem = (props: { action: Action }) => React.JSX.Element;
+/* <ActionItem /> */
+
+const ActionButton = withBaseElementProps(
+  Button,
+  ({ className = `${BLOCK_NAME}__button`, ...props }) => ({
+    ...props,
+    className,
+  })
+);
 
 interface ActionItem<T extends StorageBrowserElements = StorageBrowserElements>
   extends RenderActionItem,
-    Pick<T, 'Button' | 'ListItem' | 'Icon'> {}
```

Comment:
We should still allow for rendering an `Icon` for the `Action` I think?

Reply:
Yes, added that back in. I left out ListItem. If role="menu" is used with role="menuitem", it will read out the number of items in the menu so we don't really need it for accessibility or anything.